drivers/wiegand/wiegand.c

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright (C) 2019 Hackerspace San Salvador
+ *
+ * This driver is a port for RIOT-OS based on the Wiegand protocol for
+ * Arduino implementation from monkeyboards.org.
+ *
+ * Original implementation available at:
+ *   https://github.com/monkeyboard/Wiegand-Protocol-Library-for-Arduino
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @ingroup     drivers_wiegand
+ * @{
+ *
+ * @file
+ * @brief       Driver for the Wiegand Protocol.
+ *
+ * @author      Mario Gomez <mario.gomez@teubi.co>
+ *
+ * @}
+ */
+
+#include "xtimer.h"
+#include "wiegand_params.h"
+
+/**
+ * @brief Handler for D0 signal in Wiegand protocol
+ *
+ */
+void wg_read_d0(void *arg)
```

Comment:
private functions should be `static`.

Reply:
Done. All private functions are static now.